src/Parser/Core/Modules/Others/DistanceMoved.js

```diff
@@ -54,7 +56,13 @@ class DistanceMoved extends Analyzer {
     if (!this.lastPositionUpdate) {
       return;
     }
-    this.totalDistanceMoved += this.calculateDistance(this.lastPositionUpdate.x, this.lastPositionUpdate.y, event.x, event.y);
+    let distanceMoved = this.calculateDistance(this.lastPositionUpdate.x, this.lastPositionUpdate.y, event.x, event.y);
```

Comment:
You should try to alway make the linter happy before you create a pull request or commit :)
With `npm run lint` or `yarn run lint`

```
./src/Parser/Core/Modules/Others/DistanceMoved.js

  Line 59:  'distanceMoved' is never reassigned. Use 'const' instead  prefer-const
```

Reply:
Thank you. Corrected and Travis is happy again.